packages/react-kit/src/hooks/useCurrentSellers.ts

```diff
@@ -331,11 +338,19 @@ export function useCurrentSellers(
   const lens: Profile[] = useMemo(() => {
```

Comment:
I'm afraid the lens profile API can't be used anymore, until we migrate to LENS v2.
We need to query the on-chain data (the LENS token contract) to get the address that owns the lens tokenId.
That's doesn't use the profile data saved on the LENS api.